src/test_driver/esp32/qemu_setup.sh

```diff
@@ -34,7 +34,6 @@ die() {
 cd "$here" || die 'ack!, where am I?!?'
 
 # shellcheck source=/dev/null
-source idf.sh
 rm -f ./build/sdkconfig
-SDKCONFIG=./build/sdkconfig SDKCONFIG_DEFAULTS=sdkconfig_qemu.defaults idf make defconfig
-SDKCONFIG=./build/sdkconfig idf make -j8 esp32_elf_builder
+SDKCONFIG=./build/sdkconfig SDKCONFIG_DEFAULTS=sdkconfig_qemu.defaults idf.py build defconfig
+SDKCONFIG=./build/sdkconfig idf.py build esp32_elf_builder
```

Comment:
@andy31415 Is it possible to try this locally to confirm. If yes, are the steps captured anywhere that can help?

Reply:
For qemu, I have kept CI compilation as it is i.e using Make.

Reply:
@sweetymhaiske  Please file an isse for this so that it is tracked

Reply:
#7186 Here it is.